exercises/supervised-classification/ex_rnw/ex_research-group-logreg.Rnw

```diff
@@ -0,0 +1,48 @@
+Chapter 3.4 of the lecture introduces \textbf{logistic regression}. The medical research group from last week is eager to find out if it can be used to predict whether a patient admitted to the hospital will require intensive care. This is a binary classification task with target space $\Yspace = \setzo$, with $y=1$ if the patient requires intensive care and $y=0$ if not. The feature space is the same as before:
+$\Xspace = (\R_{0}^{+})^3$, with $\xi = (x_{age},\;x_{blood\;pressure},\;x_{weight})^{(i)} \in \Xspace$ for $i = 1, 2, \dots, n$ observations. 
+
+\medbreak
+Before the group trains a logistic regression model, researcher Holger remarks they could just as well fit a linear model (LM), as in the case of a binary classification task, both models would make identical predictions.
+
+\begin{enumerate}\bfseries
+  \item[1)] Are the predictions of a logistic regression model and a LM identical for a binary classification task? If not, explain why they could differ.
```

Comment:
- "and an LM"
- hier vlt. noch dazu, wie die predictions aussehen. Wie wäre es, sie den hyp-space hinschreiben zu lassen und ein paar Elemente des Hypspaces zeichnen zu lassen. Kann man dann geschlossen nicht mehr abfragen, aber ok.. (Oder wieder einen falschen Hyp-space und sie sollen ihn korrigieren, ja, besser so: als falschen hypspace den vom LM :) )

Reply:
Gute Idee, hab jetzt mit dem falschen Hyp-space vom LM, der korrigiert werden soll :)